gprofiler/profilers/python.py

```diff
@@ -275,7 +277,7 @@ def _select_processes_to_profile(self) -> List[Process]:
             except Exception:
                 logger.exception(f"Couldn't add pid {process.pid} to list")
 
-        return filtered_procs
+        return filtered_procs + [Process(pid) for pid in self._python_pyspy_processes]
```

Comment:
`Process(pid)` will raise if the PID exited, that's okay but let's mention in the `help` that they're expected to be alive forever